spec/prog/vnet/cert_nexus_spec.rb

```diff
@@ -114,7 +114,7 @@
 
     it "hops to restart if dns_challenge validation fails" do
       expect(challenge).to receive(:status).and_return("failed")
-      expect(Clog).to receive(:emit).with("DNS validation failed")
+      expect(Clog).to receive(:emit).with("DNS validation failed").and_call_original
```

Comment:
I wonder if we can use an rspec extension to somehow make this a do-not-forget type of item (or, I guess, a rubocop autocorrect)

Reply:
I would prefer to change `Clog.emit` to not use a block, and instead use a regular argument.  The block approach only makes sense if you were conditionally calling the block, but since `Clog.emit` always calls the block if given, using a block is only cost and no benefit.